submissions/zhenyakornilov/js-dom/js/index.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+import museums from "./data.js";
+import switchTheme from "./utils.js";
+
+function main() {
+  switchTheme();
+
+  const listMenu = document.getElementById("menu");
+  const contentBody = document.getElementById("content");
+  renderButtons(museums, listMenu);
+  toggleSelectedButton(museums, contentBody);
+  openMenu(listMenu);
+}
+
+document.addEventListener("DOMContentLoaded", main);
+
+function renderButtons(museumsObj, menu) {
+  let buttonsHTML = museumsObj
+    .map(({ btnName }) => {
+      return (
+        `<li class="nav-item">` +
+        `  <button class="btn-text">${btnName}</button>` +
+        `</li>`
+      );
+    })
+    .join("");
+
+  menu.innerHTML = buttonsHTML;
+}
+
+function toggleSelectedButton(museums, content) {
+  let selectedButton = document.getElementsByClassName("nav-item")[0];
+  selectedButton.classList.add("open");
+
+  const buttons = document.getElementsByClassName("nav-item");
+
+  function getContextObj() {
+    const selectedBtnText = document.querySelector(".open .btn-text");
+    let obj = museums.filter(
+      ({ btnName }) => btnName === selectedBtnText.textContent
+    )[0];
+
+    return obj;
+  }
+
+  renderArticle(getContextObj(), content);
+
+  for (let button of buttons) {
+    button.addEventListener("click", function () {
```

Comment:
here you added a listener to every button. why not add a listener to the navigation menu?